odl/solvers/util/callback.py

```diff
@@ -236,20 +236,25 @@ class CallbackPrintIteration(SolverCallback):
 
     _default_text = 'iter ='
 
-    def __init__(self, text=None):
+    def __init__(self, text=None, display_step=1):
         """Initialize a new instance.
 
         Parameters
         ----------
         text : string
             Text to display before the iteration count. Default: 'iter ='
+        display_step : positive int, optional
+            Number of iterations between plots. Default: 1
```

Comment:
"plots" doesn't make sense here. Use "output" or "print" or something like that.